ui/components/multichain/create-snap-account/create-snap-account.tsx

```diff
@@ -71,7 +75,32 @@ export const CreateSnapAccount = ({
   );
 
   const getNextAccountName = async () => {
-    return getNextAvailableAccountName(KeyringTypes.snap);
+    const defaultSnapAccountName = await getNextAvailableAccountName(
+      KeyringTypes.snap,
+    );
+
+    // FIXME: This is a temporary workaround to suggest a different account name for a first party snap.
+    const accountNumber = defaultSnapAccountName.split(' ').pop();
```

Comment:
Very nit: We shouldn't have a trailing whitespace, but until we remove that code, we could also `trim` here:

```suggestion
    const accountNumber = defaultSnapAccountName.trim().split(' ').pop();
```